SPECS-CROSS/gcc-bootstrap_cross/090_all_pr55930-dependency-tracking.patch

```diff
@@ -0,0 +1,18 @@
+libatomic build failure if configured with --disable-dependency-tracking
```

Comment:
Was this patch needed to fix a build error in our gcc bootstrap phase 1, 2, and 3? Adding @vitong so he is aware.

Reply:
It looks like this patch is also used in the native gcc SPEC file. From the gnu bug link below, it seems like the bug is fixed in GCC 10 (both 10.1 and 10.2 have the fix, though I haven't tested it). We can help port the fix back to GCC 9 if that's necessary